pytorch/pytorch_edgeml/graph/rnn.py

```diff
@@ -1049,7 +1049,7 @@ def forward(self, input, hiddenState=None, cellState=None):
 
 class SRNN2(nn.Module):
 
-    def __init__(self, inputDim, outputDim, hiddenDim0, hiddenDim1, cellType, **cellArgs):
+    def __init__(self, inputDim, outputDim, hiddenDim0, hiddenDim1, cellType, dropoutProbability, **cellArgs):
```

Comment:
@Sachin Please add different dropout probs for both layers (it makes a difference) and 2 use a default value of 1.0 for them.

```
 def __init__(self, inputDim, outputDim, hiddenDim0, hiddenDim1, cellType,
                    dropoutProb0=1.0, dropOutProb1=1.0, **cellArgs):
```